src/adjoint/outputForward/solverutils_d.f90

```diff
@@ -30,10 +30,10 @@ subroutine timestep_block_d(onlyradii)
     use constants
     use blockpointers, only : ie, je, ke, il, jl, kl, w, wd, p, pd, &
 &   rlv, rlvd, rev, revd, radi, radid, radj, radjd, radk, radkd, si, sid&
-&   , sj, sjd, sk, skd, sfacei, sfacej, sfacek, dtl, gamma, vol, vold, &
-&   addgridvelocities, sectionid
+&   , sj, sjd, sk, skd, sfacei, sfaceid, sfacej, sfacejd, sfacek, &
```

Comment:
this file is changed but no changes for the corresponding bwd version? You should double check why this is the case. Do we not create the bwd version of solverutils?

Reply:
will check